src/integration-test/resources/db/JurorRecordController_searchForJurorRecords.sql

```diff
@@ -19,7 +20,8 @@ VALUES ('641600091', 'MR','LNAMEFIVEFOURZERO','FNAMEFIVEFOURZERO', '1989-03-31',
 ('641500098', 'MR','Lnamenineeight','Fnamenineeight', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true),
 ('641500099', 'MR','Lnameninenine','Fnameninenine', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true),
 ('641500100', 'MR','Lnamenineten','Fnamenineten', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true),
-('641500101', 'MR','Lnamenineten','Fnamenineten', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true);
+('641500101', 'MR','Lnamenineten','Fnamenineten', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true),
+('641700120', 'MR','Lnametwo','Fnametwo I', '1988-01-01', '543 STREET NAME', 'ANYTOWN', 'CH1 2AN', true);
```

Comment:
Will need another test case for a surname with a space in it

Reply:
This gets a bit messy, the issue raised is the users dont want to have to enter the initial so will cover that in the updated code.